### PR TITLE
Remove jQuery AJAX from the notice selection deletion button

### PR DESCRIPTION
**Pull request by yardenshoham**
- Removed all jQuery AJAX calls and replaced with our fetch wrapper
- Tested the repo notice selection deletion button functionality and it works as before

# Demo using `fetch` instead of jQuery AJAX
action

### CODE REVIEW

File: web_src/js/features/admin/common.js [hunk 1]
```diff
@@ -204,22 +205,18 @@ export function initAdminCommon() {
...
+    $('#delete-selection').on('click', async function (e) {
       e.preventDefault();
       const $this = $(this);
       $this.addClass('loading disabled');
-      const ids = [];
+      const data = new FormData();
       $checkboxes.each(function () {
         if ($(this).checkbox('is checked')) {
-          ids.push($(this).data('id'));
+          data.append('ids[]', $(this).data('id'));
```
Comment (delvh) [inline web_src/js/features/admin/common.js:215]: ?
That looks really weird.
Are you sure that's the correct key?

Comment (yardenshoham) [reply web_src/js/features/admin/common.js:215]: Yes, you can see it in your devtools

Comment (delvh) [reply web_src/js/features/admin/common.js:215]: My question was rather:
The backend accepts and wants this specific key?
It does look quite strange to me…

Comment (yardenshoham) [reply web_src/js/features/admin/common.js:215]: It does
https://github.com/go-gitea/gitea/blob/f9207b09479df964872d68842469991042b5497f/routers/web/admin/notice.go#L50

**Review by delvh — APPROVED** [review]
Interesting that we expect something so unnatural in the backend…

**Comment by wxiaoguang** [comment]
> Interesting that we expect something so unnatural in the backend…

That's natural for many backends, eg: PHP: https://stackoverflow.com/questions/1763508/passing-arrays-as-url-parameter

**Review by wxiaoguang — APPROVED** [review]

**Pull request merged**